tar/ccache/ccache_write.c

```diff
@@ -83,6 +87,10 @@ callback_write_rec(void * cookie, uint8_t * s, size_t slen, void * rec)
 	if (ccr->age > MAXAGE)
 		goto done;
 
+	/* Don't write an entry if it has negative mtime. */
```

Comment:
We should check this before the assertions, since if `sizeof(uintmax_t) > sizeof(uint64_t)` the assertion will still fail for an mtime of -1.